sdk/go/x/auto/stack.go

```diff
@@ -592,6 +593,29 @@ type UpResult struct {
 	Summary UpdateSummary
 }
 
+// GetPermalink returns the permalink URL in the Pulumi Console for the update
+// or refresh operation.
```

Comment:
Might be good to add that for stacks using local backends this function will return an error (I think?).